4-tools/presentation.ipynb

```diff
@@ -475,33 +486,36 @@
     "It can fix some issues automatically and has several editor integrations. To run it (once installed), use the following command.\n",
     "\n",
     "`ruff check is_this_pep8.py`\n",
-    "##### Run in terminal and compare results.\n",
-    "\n"
+    "##### Run in terminal.\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
+    "editable": true,
     "slideshow": {
      "slide_type": "slide"
-    }
+    },
+    "tags": []
    },
    "source": [
     "# [`black`](https://pypi.org/project/black/)\n",
     "\n",
-    "`black` is an automatic code formater. When `black` is run on a script it will change to code into its style, taking the decision away from the user. This means that code formatted with `black` has a very consistent look.\n",
+    "`black` is an automatic code formater. When `black` is run on a script it will change the code into its style, taking the decision away from the user. This means that code formatted with `black` has a very consistent look.\n",
     "\n",
     "All the code demonstrated in this course has been formtated with `black` (except for the deliberately bad examples). \n",
     "\n",
-    "Automatic code formaters like `black` are very useful for collaborative projects. It is already used by pytest, pandas, Django and Astropy (WIP) and Dropbox, Mozilla, Duolingo, Facebook etc. "
+    "Automatic code formaters like `black` are very useful for collaborative projects. It is already used by pytest, pandas, Django, Astropy, Dropbox, Mozilla, Duolingo, Facebook etc. "
```

Comment:
I suspect quite a few of these might switch to Ruff formatter, so this list needs to be checked both this year and next year.

Reply:
I checked and as far as I can tell this list is still true. Will need to check again for next year.